src/autocomplete.js

```diff
@@ -520,6 +525,9 @@ export default class Autocomplete extends Component {
             <li className={`${optionClassName} ${optionClassName}--no-results`}>{tNoResults()}</li>
           )}
         </ul>
+
+        <span id='assistiveHint' style={{ display: 'none' }}>When autocomplete results are available use up and down arrows to review and enter to select.  Touch device users, explore by touch or with swipe gestures.</span>
```

Comment:
This will need to be translatable, like `tNoResults`.

Reply:
Acknowledged - will revise.

Reply:
Moved into translatable props, and README updated.